app/uk/gov/hmrc/gform/gformbackend/GformConnector.scala

```
@@ -26,7 +26,7 @@ import scala.concurrent.{ ExecutionContext, Future }
 
 @Singleton
 class GformConnector @Inject()(ws: GformWSHttp, servicesConfig: ServicesConfig) {
-  lazy val baseUrl = servicesConfig.baseUrl("gform") + "/gform"
+  lazy val baseUrl = servicesConfig.baseUrl("gform") + servicesConfig.getConfString("gform.path-prefix", "")
```

I'm guessing the second argument is a default. Wouldn't this cause silent issues? If that's the case. I think it'd be better to throw an exception or have a sensible default for dev envs.

---

NABD could use this instead: https://github.com/hmrc/bootstrap-play-26/blob/master/src/main/scala/uk/gov/hmrc/play/bootstrap/config/ServicesConfig.scala#L77

But Alex has explained offline that it's no big deal in practice. So I'll leave it up to you. Approving.